client/src/views/Simulation/components/RunButton.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="btn-group">
+    <button
+      type="button"
+      class="btn btn-primary"
+      :class="{ 'blue' : !autoRunInput }"
+      @click="onClickRun"
+    >
+      <font-awesome-icon :icon="['fas', ( autoRunInput ? 'pause' : 'play') ]" />
+      {{ displayText }}
+    </button>
+    <button
+      type="button"
+      class="btn btn-primary blue dropdown-toggle dropdown-toggle-split"
+      @click="dropdownOpen = !dropdownOpen"
+    />
+    <div class="dropdown-menu dropdown-menu-right" :class="{ 'show': dropdownOpen }">
+      <input type="checkbox" v-model="autoRunInput" id="auto-run-input" />
+      <label
+        for="auto-run-input"
+        title="Enable model to run automatically on when Parameters are updated">
+        Auto-Run
+      </label>
+      <div class="dropdown-divider" />
+      <h6 class="dropdown-header">_Sequence_ Settings</h6>
```

Comment:
`Model Execution Settings` instead I'd say.

Reply:
Perfect. That's why I added the `_` to be sure to found a better name.